src/stories/buttons/button/index.tsx

```diff
@@ -2,29 +2,6 @@ import { forwardRef } from "react";
 import { Button as ZendeskButton } from "@zendeskgarden/react-buttons";
 import styled from "styled-components";
 import { ButtonArgs } from "./_types";
-import { theme } from "../../theme";
-
-let customTheme = {
-  ...theme,
-};
-
-// Define theme override
-const getThemeStyle = (props: ButtonArgs) => {
-  const { themeColor } = props;
-  if (themeColor) {
-    customTheme = {
```

Comment:
questa parte era per gestire temi (colori) diversi sui bottoni, non so se la leverei (potrebbe tornare utile in futuro)